src/plugins/intel_gpu/include/intel_gpu/primitives/convolution.hpp

```diff
@@ -530,6 +530,47 @@ struct convolution : public primitive_base<convolution> {
           activations_zero_points(std::vector<primitive_id>(0)),
           compensation(std::vector<primitive_id>(0)) {}
 
+    /// @brief Constructs convolution primitive (w/o bias).
+    /// @param id This primitive id.
+    /// @param input Input primitive id.
+    /// @param weights List of primitive ids containing weights data.
+    /// @param groups Number of filter groups.
+    /// @param pad Defines logical pad value added to input tensor
+    /// @param stride Defines shift in input buffer between adjacent calculations of output values.
+    /// @param dilation Defines gaps in the input - dilation rate k=1 is normal convolution,
+    /// k=2 means skipping one pixel per input, k=4 means skipping 3 pixels.
+    /// As an example in one dimension, a filter w of size 3 would compute over input x the following:
+    /// w[0]*x[0] + w[1]*x[1] + w[2]*x[2] for dilation of 1.
+    /// For dilation 2 the filter would instead compute w[0]*x[0] + w[1]*x[2] + w[2]*x[4].
+    /// @param output_size User-defined output data size of the primitive (w/o padding).
+    convolution(const primitive_id& id,
+                const primitive_id& input,
+                const std::vector<primitive_id>& weights,
+                uint32_t groups,
+                ov::Strides stride,
+                ov::CoordinateDiff pad,
+                ov::Strides dilation,
+                tensor output_size,
+                bool grouped_weights_shape = false,
+                const primitive_id& ext_prim_id = "",
+                const padding& output_padding = padding())
```

Comment:
I think we can add output_size only with a default value.

Reply:
done